python/example_code/medical-imaging/medical_imaging_basics.py

```diff
@@ -441,4 +461,181 @@ def list_tags_for_resource(self, resource_arn):
+        print(f"Copied image set to new image set with ID : {copied_image_set_id}")
+
+        image_set_ids.append(copied_image_set_id)
+
+        # Wait for copied image set to be ACTIVE before updating the metadata.
+        while True:
+            time.sleep(1)
+            try:
+                image_set_properties = self.get_image_set(data_store_id, copied_image_set_id)
+            except ClientError as err:
+                print(f"get_image_set raised an error {err.response['Error']['Message']}")
+                break
+
+            image_set_state = image_set_properties['imageSetState']
+            print(f'Image set with id : "{copied_image_set_id}" has status: "{image_set_state}"')
+            if image_set_state != "LOCKED":
+                break
+
+        attributes = "{\"SchemaVersion\":1.1,\"Patient\":{\"DICOM\":{\"PatientName\":\"Garcia^Gloria\"}}}"
+        metadata = {"DICOMUpdates": {"updatableAttributes": attributes}}
+
+        self.update_image_set_metadata(data_store_id, copied_image_set_id, "1", metadata)
+        print(f"Updated metadata for image set with id : {copied_image_set_id}")
+
+        # Wait for all image sets to change from LOCKED status before deleting.
+        for image_set_id in image_set_ids:
+            while True:
+                time.sleep(1)
+                try:
+                    image_set_properties = self.get_image_set(data_store_id, image_set_id)
+                except ClientError as err:
+                    print(f"get_image_set raised an error {err.response['Error']['Message']}")
+                    break
+
+                image_set_state = image_set_properties['imageSetState']
+                print(f'Image set with id : "{image_set_id}" has status: "{image_set_state}"')
+                if image_set_state != "LOCKED":
+                    break
+
+        for image_set_id in image_set_ids:
+            self.delete_image_set(data_store_id, image_set_id)
+            print(f"Deleted image set with id : {image_set_id}")
+
+        # Wait for image sets to be deleted before deleting the data store.
+        for image_set_id in image_set_ids:
+            while True:
+                time.sleep(1)
+                try:
+                    image_set_properties = self.get_image_set(data_store_id, image_set_id)
+                except ClientError as err:
+                    print(f"get_image_set raised an error {err.response['Error']['Message']}")
+                    break
+
+                image_set_state = image_set_properties['imageSetState']
+                print(f'Image set with id : "{image_set_id}" has status: "{image_set_state}"')
+                if image_set_state == "DELETED":
+                    break
+
+        self.delete_datastore(data_store_id)
+        print(f"Data store deleted with id : {data_store_id}")
+
+
+if __name__ == '__main__':
+    # Replace these values with your own.
+    source_s3_uri = "s3://medical-imaging-dicom-input/dicom_input/"
```

Comment:
Should you include this in the custom instructions in the README? I didn't see instructions on how to run this.